cmd/podman/system/connection/add.go

```diff
@@ -236,77 +279,37 @@ func getUDS(cmd *cobra.Command, uri *url.URL) (string, error) {
 			}
 		}
 	}
-
 	var authMethods []ssh.AuthMethod
 	if len(signers) > 0 {
 		var dedup = make(map[string]ssh.Signer)
-		// Dedup signers based on fingerprint, ssh-agent keys override CONTAINER_SSHKEY
 		for _, s := range signers {
 			fp := ssh.FingerprintSHA256(s.PublicKey())
 			if _, found := dedup[fp]; found {
-				logrus.Debugf("Dedup SSH Key %s %s", ssh.FingerprintSHA256(s.PublicKey()), s.PublicKey().Type())
+				logrus.Debugf("Dedup SSH Key %s %s", ssh.FingerprintSHA256(s.PublicKey()), s.PublicKey().Type()) // public key info and validation
 			}
 			dedup[fp] = s
 		}
-
 		var uniq []ssh.Signer
 		for _, s := range dedup {
 			uniq = append(uniq, s)
 		}
-
 		authMethods = append(authMethods, ssh.PublicKeysCallback(func() ([]ssh.Signer, error) {
 			return uniq, nil
 		}))
 	}
-
-	if passwdSet {
+	if passwdSet { // proper password validation for ssh connection if necessary
```

Comment:
I think `s/validation/authentication/`? But I’m also not quite sure what the comment is trying to point out.